suite/tests/client-interface/cleancall.dll.c

```diff
@@ -349,6 +367,10 @@ bb_event(void *drcontext, void *tag, instrlist_t *bb, bool for_trace, bool trans
 #endif
     }
 
+    /* Inlineable call (cleancall-opt does further tests there). */
+    dr_insert_clean_call(drcontext, bb, instrlist_first(bb), inc_counter, false, 1,
```

Comment:
Not clear what the rationale of this clean call is. What is it trying to test?

Reply:
Yeah I was trying to replicate a failure that the inscount sample was hitting, but somehow I couldn't reproduce it here, even with this call which looked identical.  Not sure why, but I fixed it, and we do have sample tests too.  I could just remove it -- sure, will revert this.